composer.json

```diff
@@ -24,44 +24,51 @@
         "issues": "https://github.com/FriendsOfSymfony/FOSCommentBundle/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.5.9|~7.0",
         "friendsofsymfony/rest-bundle": "~1.6|~2.0",
-        "jms/serializer-bundle": "~0.11|~1.0",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/asset": "~2.3|~3.0",
-        "symfony/dependency-injection": "~2.3|~3.0",
-        "symfony/routing": "~2.3|~3.0",
-        "symfony/security-bundle": "~2.3|~3.0",
-        "symfony/yaml": "~2.3|~3.0",
-        "symfony/form": "~2.3|~3.0",
-        "symfony/validator": "~2.3|~3.0",
-        "symfony/twig-bundle": "~2.3|~3.0",
-        "symfony/twig-bridge": "^2.3.4|~3.0",
-        "twig/twig": "~1.28|~2.0"
+        "jms/serializer-bundle": "~1.0",
+        "symfony/framework-bundle": "~2.7|~3.0",
+        "symfony/asset": "~2.7|~3.0",
+        "symfony/dependency-injection": "~2.7|~3.0",
+        "symfony/routing": "~2.7|~3.0",
+        "symfony/security-bundle": "~2.7|~3.0",
+        "symfony/yaml": "~2.7|~3.0",
+        "symfony/form": "~2.7|~3.0",
+        "symfony/validator": "~2.7|~3.0",
+        "symfony/twig-bundle": "~2.7|~3.0",
+        "symfony/twig-bridge": "^2.7|~3.0",
+        "twig/twig": "~1.28|~2.0",
+        "phpoption/phpoption": "~1.1",
+        "doctrine/annotations": "~1.0"
     },
     "suggest": {
         "friendsofsymfony/user-bundle": "Allows for user integration.",
         "ornicar/akismet-bundle": "Integrate Akismet for spamdetection."
     },
+    "conflict": {
+        "twig/twig": "<1.28"
+    },
     "require-dev": {
-        "symfony/assetic-bundle": "~2.3",
-        "symfony/browser-kit": "~2.3|~3.0",
-        "symfony/dom-crawler": "~2.1|~3.0",
-        "symfony/css-selector": "~2.3|~3.0",
+        "symfony/assetic-bundle": "~2.7",
+        "symfony/browser-kit": "~2.7|~3.0",
+        "symfony/dom-crawler": "~2.7|~3.0",
+        "symfony/css-selector": "~2.7|~3.0",
         "doctrine/orm": "~2.3",
-        "doctrine/doctrine-bundle": "~1.1",
-        "friendsofsymfony/user-bundle": "~1.1|~2.0@dev",
+        "doctrine/doctrine-bundle": "~1.6",
+        "friendsofsymfony/user-bundle": "~1.3|~2.0@dev",
         "ornicar/akismet-bundle": "dev-master",
-        "symfony/expression-language": "~2.4|~3.0",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/expression-language": "~2.7|~3.0",
+        "symfony/phpunit-bridge": "~2.7|~3.0",
+        "phpunit/phpunit": "~4.8|~5.0"
     },
     "extra": {
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
     },
     "autoload": {
-        "psr-0": { "FOS\\CommentBundle": "" }
+        "psr-4": { "FOS\\CommentBundle\\": "" },
```

Comment:
this change is wrong, as you forgot to remove the ``target-dir``, making the composer.json invalid